multicluster/cmd/multicluster-controller/controller.go

```diff
@@ -18,7 +18,7 @@ import (
 	"fmt"
 	"time"
 
-	// Import all Kubernetes client auth plugins (e.g. Azure, GCP, OIDC, etc.)
+	// Import all Kubernetes client auth plugins (e.g. GCP, OIDC, etc.)
```

Comment:
Not really needed as we still technically load the Azure auth plugin I think :)

Reply:
Got it. Reverted this change.